grails-app/domain/org/pih/warehouse/shipping/ShipmentItem.groovy

```diff
@@ -97,6 +97,11 @@ class ShipmentItem implements Comparable, Serializable {
         return orderItemIds ? orderItemIds.first() : null
     }
 
+    String getOrderId() {
+        def orderNumbers = orderItems?.collect { OrderItem orderItem -> orderItem.order.id }?.unique()
```

Comment:
I know we did that before, but perhaps it would be better to pull an associated order? Could be refactored in near future?

Reply:
cc @jmiranda

Reply:
This could return multiple order IDs, so you'd have to retrieve first/last or allow returning a list.

Reply:
Ah nevermind didn't see the next line. Yeah returning the order would probably be better but I'd have to see the context in which this method is used.